client-side/src/Components/studentComponent/Gatepasses/LocalFixed/Form/LFform.js

```diff
@@ -140,11 +140,33 @@ const LFform = (props) => {
     const res3 = await checkBlacklist();
     const res4 = await checkApprovedOrCheckedout();
 
-    if (res1 == true && res2 < props.weekLimit && res3 == false && res4 == 0) {
-      return true;
-    } else {
+    if (res3 == true) {
+      alert("Cannot Apply: You are blacklisted, you cannot apply for Gatepass");
+      return false;
+    } else if (res2 >= props.weekLimit) {
+      alert(
+        "Cannot Apply: You have exhausted all your weekly Local Fixed Gatepass "
+      );
+      return false;
+    } else if (res4 != 0) {
+      alert(
+        "Cannot Apply: You have already applied for a Local Fixed Gatepass"
+      );
+      return false;
+    } else if (res1 == false) {
+      alert(
+        "Cannot Apply: You are not within the time limit of Local FIxed Gatepass"
```

Comment:
1) The "I" in Fixed should be small
2) I think it would be better if "You cannot apply a gatepass in the outside hours"